src/backend/src/prisma/schema.prisma

```diff
@@ -636,3 +653,42 @@ model Meeting {
   team         Team?  @relation(fields: [teamId], references: [teamId])
   teamId       String
 }
+
+model Wbs_Proposed_Changes {
+  proposedChangeId     String                         @id @default(uuid())
+  name                 String
+  status               WBS_Element_Status
+  projectLeadId        Int
+  projectManagerId     Int
+  deletedByUserId      Int
+  links                Link[]                         @relation(name: "proposedChangeLinks")
+  changRequestId       Int                            @unique
+  projectChanges       Project_Proposed_Changes?      @relation(name: "proposedProjectChanges", fields: [projectChangesId], references: [proposedChangeId])
+  projectChangesId     String?                        @unique
+  workPackageChanges   Work_Package_Proposed_Changes? @relation(name: "proposedWorkPackageChanges", fields: [workPackageChangesId], references: [proposedChangeId])
+  workPackageChangesId String?                        @unique
+}
+
+model Project_Proposed_Changes {
+  proposedChangeId  String                @id @default(uuid())
+  budget            Int
+  summary           String
+  rules             String[]
+  goals             Description_Bullet[]  @relation(name: "proposedProjectGoals")
+  features          Description_Bullet[]  @relation(name: "proposedProjectFeatures")
+  otherConstraints  Description_Bullet[]  @relation(name: "proposedProjectOtherConstraints")
+  teams             Team[]                @relation(name: "proposedProjectTeams")
+  proposedWbsChange Wbs_Proposed_Changes? @relation(name: "proposedProjectChanges")
```

Comment:
this shouldn't be optional, every proposed project changes will have a proposed wbs changes, but not vice versa

Reply:
also this should be proposedWbsChanges